projects/objectnav_baselines/experiments/objectnav_mixin_ddppo.py

```diff
@@ -26,6 +69,54 @@ def training_pipeline(self, **kwargs):
         use_gae = True
         gae_lambda = 0.95
         max_grad_norm = 0.5
+
+        # auxliary losses
+        aux_loss_total_weight = 2.0
+
+        # Total losses
+        total_aux_losses: Dict[str, Tuple[AbstractActorCriticLoss, float]] = {
+            InverseDynamicsLoss.UUID: (
+                InverseDynamicsLoss(
+                    subsample_rate=0.2, subsample_min_num=10,  # TODO: test its effects
+                ),
+                0.05 * aux_loss_total_weight,  # should times 2
+            ),
+            TemporalDistanceLoss.UUID: (
+                TemporalDistanceLoss(
+                    num_pairs=8, epsiode_len_min=5,  # TODO: test its effects
+                ),
+                0.2 * aux_loss_total_weight,  # should times 2
+            ),
+            CPCA1Loss.UUID: (
+                CPCA1Loss(subsample_rate=0.2,),  # TODO: test its effects
+                0.05 * aux_loss_total_weight,  # should times 2
+            ),
+            CPCA2Loss.UUID: (
+                CPCA2Loss(subsample_rate=0.2,),  # TODO: test its effects
+                0.05 * aux_loss_total_weight,  # should times 2
+            ),
+            CPCA4Loss.UUID: (
+                CPCA4Loss(subsample_rate=0.2,),  # TODO: test its effects
+                0.05 * aux_loss_total_weight,  # should times 2
+            ),
+            CPCA8Loss.UUID: (
+                CPCA8Loss(subsample_rate=0.2,),  # TODO: test its effects
+                0.05 * aux_loss_total_weight,  # should times 2
+            ),
+            CPCA16Loss.UUID: (
+                CPCA16Loss(subsample_rate=0.2,),  # TODO: test its effects
+                0.05 * aux_loss_total_weight,  # should times 2
+            ),
+        }
+        named_losses = {uuid: total_aux_losses[uuid] for uuid in cls.AUXILIARY_UUIDS}
+        named_losses["ppo_loss"] = (PPO(**PPOConfig), 1.0)
+
+        if cls.multiple_beliefs:  # add weight entropy loss automatically
+            named_losses[TaskWeightsLoss.UUID] = (
+                TaskWeightsLoss(cls.AUXILIARY_UUIDS),
+                0.01,
+            )
```

Comment:
Can we refactor this into another function so that we can do something like:
```python
named_loses = {"ppo_loss": (PPO(**PPOConfig), 1.0)}
self._update_with_auxiliary_losses(named_loses)
```